### PR TITLE
PMM-7775 fix UI tests

### PR DESCRIPTION
**Pull request by IvanaBrecherova**

### CODE REVIEW

**Comment by IvanaBrecherova** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/7563/pipeline

File: pmm-app/tests/pages/pmmSettingsPage.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const { I } = inject();
 const assert = require('assert');
+const adminPage = require('./adminPage');
```
Comment (puneet0191) [inline pmm-app/tests/pages/pmmSettingsPage.js:3]: could you please use inject above for using the adminPage object?

File: pmm-app/tests/pages/remoteInstancesPage.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ module.exports = {
   },
 
   tableStatsLimitRadioButtonLocator(limit) {
-    return `//label[@for='${limit}']`;
+    return `//label[contains(text(), '${limit}')]`;
```
Comment (puneet0191) [inline pmm-app/tests/pages/remoteInstancesPage.js:68]: you can now use locate for this.

**Review by puneet0191 — CHANGES_REQUESTED** [review]

**Review by yurkovychv — APPROVED** [review]

**Comment by IvanaBrecherova** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/7571/pipeline

**Review by puneet0191 — APPROVED** [review]

**Review by lunaticusgreen — APPROVED** [review]

**Pull request merged**